packages/react-components/react-migration-v8-v9/src/components/Checkbox/CheckboxShim.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { classNamesFunction, ICheckboxProps, ICheckboxStyles, ICheckboxStyleProps } from '@fluentui/react';
+import { Checkbox, mergeClasses } from '@fluentui/react-components';
+import { useControllableValue } from '@fluentui/react-hooks';
+import { shimCheckboxProps } from './shimCheckboxProps';
+import { useCheckboxStyles } from './Checkbox.styles';
+
+const getClassNames = classNamesFunction<ICheckboxStyleProps, ICheckboxStyles>({
+  useStaticStyles: true,
+});
+
+export const CheckboxShim = (props: ICheckboxProps) => {
```

Comment:
can we add test that this control works as expected ?

Reply:
Yes, already added the unit test: https://github.com/microsoft/fluentui/blob/ddbaf4dd5b8a5e3de9fd68ed35f11d2d5a77f5fe/packages/react-components/react-migration-v8-v9/src/components/Checkbox/CheckboxShim.test.tsx

Reply:
> can we add test that this control works as expected ?

Yes, already added the unit test: https://github.com/microsoft/fluentui/blob/ddbaf4dd5b8a5e3de9fd68ed35f11d2d5a77f5fe/packages/react-components/react-migration-v8-v9/src/components/Checkbox/CheckboxShim.test.tsx